sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,291 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
+
+### 5. Favorite Page
+
+The Favorite page will display a list of all favorite art pieces. Each art piece will show its image, artist’s name, title, and a favorite button.
+
+Users should be able to save their favorite art pieces both on the Gallery page and the Detail page.
+
+To achieve this, we’ll divide the tasks into two parts:
+
+1. Implement the favorite button where needed.
+2. Create the Favorite page.
+
+### 5.1. Favorite Button
+
+Users want the ability to add additional information to each art piece. Consider where to store the art piece data to ensure all pages can access it efficiently. Remember that we’ll be reusing this favorite button across multiple pages.
+
+<details>
+<summary>Hint ✨</summary>
+
+In Next.js, the recommended place to store data or states that need to be accessed across the app is in `_app.js`.
+
+It’s best practice to consolidate related information into a single state. However, if you're not comfortable with this approach, you can use separate states. Just ensure you can clearly identify which information corresponds to each art piece—consider saving a unique identifier, such as a slug, within each state for clarity.
+
+**Example:**
+
+- **Using a Single State:**
+
+   ```js
+   const [carInfo, setCarInfo] = useState([
+      { model: "vw-taos", colors: ["red", "blue", "white"], automatic: true }, 
+      { model: "vw-golf", colors: [ "black", "white"], automatic: false }
+      ]);
+   ```
 
----
+- **Separate States per information:**
+
+   ```js
+   const [carColors, setCarColors] = useState([
+      { model: "vw-taos", colors: "red"},
+      { model: "vw-taos", colors: "blue" }, 
+      { model: "vw-golf", colors: "black" }
+      ]);
+   
+   const [carAutomatic, setCarAutomatic] = useState([
+       "vw-taos", 
+       "vw-golf",
+      ]);
+   ```
 
-### User Story 5: Favorites
+</details>
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** mark art piece pieces as favorites
+**I want to** mark art pieces as favorites
 
-**so that** I can find them easier.
+**so that** I can find them more easily.
 
 #### Acceptance Criteria
 
-- [ ] The favorite-button is displayed in each entry in the list view
-- [ ] The favorite-button is displayed in the details view
-- [ ] The favorite-button is NOT displayed in the spotlight view
-- [ ] Clicking the favorite-button on a non-favorite piece saves it as a favorite
-- [ ] Clicking the favorite-button on a favorite piece removes it from favorites
+- [ ] The favorite button is displayed on each entry in the list view.
+- [ ] The favorite button is displayed in the details view.
+- [ ] The favorite button is displayed in the spotlight view.
+- [ ] Clicking the favorite button on a non-favorite piece saves it as a favorite.
+- [ ] Clicking the favorite button on a favorite piece removes it from favorites.
+- [ ] The favorite state of an art piece is visually highlighted by changing the color of the favorite button.
```

Comment:
At the risk of being super annoying....

I think these criteria could be improved because ssers don’t see the technical action of saving a favorite, they just see a visual change (like color or icon). 

Therefore I suggest to focus on the visible feedback.


```suggestion
- [ ] Clicking the favorite button on a non-favorite piece, it visibly indicates that the piece is now a favorite (e.g., by changing color or icon).
- [ ] Clicking the favorite button on a favorite piece, it visibly indicates that the piece is no longer a favorite.
- [ ] Favorite art pieces are visually highlighted, clearly showing their favorite status.
```